app/apps.json

```diff
@@ -205,5 +205,32 @@
       "type": "addon",
       "url": "https://github.com/xeonchen/fxos-line-fixer",
       "revision": "4c93314c2fa985aa41316f9423f06090dec059d4"
+    },
+    "https://lmorchard.github.io/fxos-addon-messages-bigger-send-button/manifest.webapp": {
```

Comment:
Thanks @lmorchard!! I know this might be asking a lot, but is there any chance I can convince you to just submit your changes for the bigger send button directly to Gaia? I can't think of any reason we would not want this, especially since the z3c is now the officially supported device (over flame).